lms/templates/fields/field_dropdown_account.underscore

```diff
@@ -20,6 +20,13 @@
         <span class="u-field-value-readonly"></span>
     <% } else { %>
         <select name="select" id="u-field-select-<%- id %>" aria-describedby="u-field-help-message-<%- id %>">
+            <% if (preOptions) { %>
+                <optgroup label="Common Time Zones">
+                <% _.each(preOptions, function(time_zone) { %>
+                   <option value="<%- time_zone.time_zone %>"><%- time_zone.description %></option>
+                <% }); %>
+                <optgroup label="All Time Zones">
+            <% } %>
             <% if (showBlankOption) { %>
                 <option value=""></option>
             <% } %>
```

Comment:
The blank option (if specified) should come before the preOptions, as its purpose is to make the default choice no choice.